mautic/mautic
# Role export access in Leads, Forms, Reports

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 


#### Description:
Added new permission for export in form results. This permission give ability enable/disable export permission for Contacts, Forms, Reports.

![image](https://github.com/mautic/mautic/assets/462477/d8cbf26b-b0a5-4ba7-94a3-d87ebf5eee72)


#### Steps to test this PR:
1.  Create new user with role and enable all permission for Contacts, Forms, Reports with checked  new Export access: Enable
2.  Login as new user and see If exports buttons exists and exports works properly in all sections: Contacts (batch export), Form (results exports) and Reports
3. Open HTML exports of Form results and Reports.
4. Edit in incognito user permission and disable export 
5. See If the export button are hidden in Form results and  Reports and Contacts lists in batch.
6. See If opened HTML export from 3. Return access denied after refresh
